bears/general/LineCountBear.py

```diff
@@ -18,24 +19,58 @@ def _get_blank_line_count(self, file):
         return num_blank_lines
 
     def run(self, filename, file, max_lines_per_file: int,
-            exclude_blank_lines: bool=False):
+            ignore_files_regex: typed_list(str),
+            exclude_blank_lines: bool=False,
+            min_lines_per_file: int=1):
         """
-        Count the number of lines in a file and ensure that they are
-        smaller than a given size.
+        Count the number of lines in a file and ensure that they lie within
+        the range of given sizes. There are files for which lower limit isn't
+        required, but these files still shouldn't be rejected. Some of such
```

Comment:
if a limit is not required, then the files won't be rejected all times, right? This is kinda confusing...

Reply:
@Makman2 There are files which are below lower limit, but the files are still needed in project. For example init.py files.
Maybe I should set min_lines_per_file to 0 and I won't mention any exceptions? Then user can define his own exceptions, if user will have bigger minimum limit than 0 ? :)

Reply:
`min_lines_per_file` should be disabled by default (--> `=0`), but I rather meant to improve the phrasing here :)